addons/core/functions/fnc_settingForceArray.sqf

```diff
@@ -0,0 +1,46 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_settingForceArray
+
+    Author(s):
+        Dorbedo
+
+    Description:
+        Forces the input into an array format and removes invalid input
+
+    Parameters:
+    0: STRING - frequencies input
```

Comment:
Indentation wrong. If you don't care I'll push anyway. Just trying to trigger your OCD.

Reply:
well done.... next time trigger your own